optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/rostergrid/powers/CollisionFreeSubLaneFactory.java

```diff
@@ -37,7 +37,7 @@
         return blobs.sorted(comparing(Blob::getPositionInGridPixels))
                 .map(this::singletonSubLaneList)
                 .reduce(this::merge)
-                .orElseGet(Collections::emptyList);
+                .orElseGet(() -> new ArrayList<>(Arrays.asList(new SubLane<>())));
```

Comment:
Why did you have to make this list mutable?

Reply:
Needs to be, you are removing elements from it in https://github.com/kiegroup/optashift-employee-rostering/blob/2ebc8b5964a4eef7bb38598d917e4939af3b9d5a/optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/rostergrid/list/ListView.java#L53